Backend/dev/migrations/dynamic-offer-driver-app/0425-intercity-special-zone.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO atlas_driver_offer_bpp.fare_product(
```

Comment:
Please mention properly merchant_id for whatever merchant you are adding. Maybe join from Merchant table and filter using shortId.